website/docs/help-and-support/troubleshooting-guide/deployment-errors.mdx

```diff
@@ -230,4 +230,36 @@ Run the restart command once again:
 If you haven't been using your Appsmith instance for a while, it could be possible that there might be multiple versions that show up in the error message. In such a case, perform the same steps for each version in order, starting from the oldest.
 :::
 
-To prevent similar issues in the future, consider [enabling auto updates](/getting-started/setup/installation-guides/docker#enabling-appsmith-auto-updates) for your Appsmith instance. If you already had auto updates on and still encountered this issue, please contact the [Appsmith team](https://discord.com/invite/rBTTVJp) to determine the cause of the deployment problem
\ No newline at end of file
+To prevent similar issues in the future, consider [enabling auto updates](/getting-started/setup/installation-guides/docker#enabling-appsmith-auto-updates) for your Appsmith instance. If you already had auto updates on and still encountered this issue, please contact the [Appsmith team](https://discord.com/invite/rBTTVJp) to determine the cause of the deployment problem
+
+## Docker auto update / watchtower error on arm64
+
+ ```
+docker-compose ps
+        Name                      Command                  State                                       Ports                                 
+---------------------------------------------------------------------------------------------------------------------------------------------
+appsmith               /opt/appsmith/entrypoint.s ...   Up (healthy)   0.0.0.0:443->443/tcp,:::443->443/tcp, 0.0.0.0:80->80/tcp,:::80->80/tcp
+ubuntu_auto_update_1   /watchtower --schedule 0 0 ...   Restarting    
+ ```
+#### Error message
+
+<Message
+messageContainerClassName="error" 
+messageContent="auto_update_1 | exec /watchtower: exec format error"></Message>
+
+#### Cause
+On an arm64 architecture, watchtower may encounter errors when attempting to update the containers. This can be caused by a variety of factors, such as compatibility issues with the specific version of the container image being used or issues with the configuration of the Docker environment. 
+
+#### Solution
+To resolve this issue, update the following line in the docker-compose file:
+
+ ```
+image: containrrr/watchtower:latest-dev
+to:
+image: containrrr/watchtower
+ ```
+
+After making the necessary changes, run the command ```docker-compose up -d``` again to start the containers. If you still encounter this issue, please contact the [Appsmith team](https://discord.com/invite/rBTTVJp) to determine the cause of the deployment problem
```

Comment:
Can we move the line "If you still encounter this issue, please contact the [Appsmith team](https://discord.com/invite/rBTTVJp) to determine the cause of the deployment problem" as common to the bottom of this page and avoid mentioning it after every troubleshooting issue?

Reply:
done